### ISSUE TITLE
docker cannot started

### ISSUE THREAD

**Issue by yxue26** [docker/for-mac#5591]
- [ ] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID: 487ED41D-7034-46D7-9D16-9AFBB82EC78C

### Information

  - macOS Version:  Big Sur: 11.2.2
  - Intel chip or Apple chip: Intel
  - Docker Desktop Version:  

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by yxue26** [comment]
macOS Version: 11.2.2
Intel chip
Docker Desktop Version: 3.3.

**Comment by djs55** [comment]
@yxue26 thanks for your report. Unfortunately the diagnostics id "487ED41D-7034-46D7-9D16-9AFBB82EC78C" is incomplete. Could you upload a fresh set of diagnostics and quote the full ID? It should look like this: (`<uuid>/<yyyymmddhhmmss>`)

![image](https://user-images.githubusercontent.com/198586/115238970-2cecc900-a116-11eb-9d49-596bada1775a.png)

**Comment by dhinojosa** [comment]
I got it to work (Maybe). I moved ~/.kube/config to another backup file name and it worked. All my Kubernetes clusters are down right now intentionally although I still had a config file that points to them. I think somehow it is trying to resolve what Kubernetes clusters you have running, and if it points to a cluster that doesn't exist then Docker never starts.

Hope that works for all of you. Let me know. I am interested.
